src/BizHawk.Emulation.Cores/Consoles/Nintendo/NES/NES.iNES.cs

```diff
@@ -52,11 +52,11 @@ public static bool DetectFromINES(byte[] data, out CartInfo Cart, out CartInfo C
 				// only add submapper if it is non-zero
 				if (submapper != 0)
 				{
-					CartV2.BoardType = $"MAPPER{mapper:d4}-{submapper:d2}";
+					CartV2.BoardType = $"MAPPER{mapper:d3}-{submapper:d2}";

```

Comment:
Don't change this or the lines below, you will break existing working identifications